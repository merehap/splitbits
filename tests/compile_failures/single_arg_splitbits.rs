use splitbits::*;

fn main() {
    splitbits!(0b11011101);
    splitbits!("aaabbccc");
}
