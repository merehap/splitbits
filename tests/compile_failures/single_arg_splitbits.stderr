error: proc macro panicked
 --> tests/compile_failures/single_arg_splitbits.rs:4:5
  |
4 |     splitbits!(0b11011101);
  |     ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: splitbits must take at least two arguments: an input value then a template. Found:
          `0b11011101`

error: proc macro panicked
 --> tests/compile_failures/single_arg_splitbits.rs:5:5
  |
5 |     splitbits!("aaabbccc");
  |     ^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: splitbits must take at least two arguments: an input value then a template. Found:
          `"aaabbccc"`
